src/rpc/rawtransaction.cpp

```diff
@@ -1203,7 +1214,12 @@ UniValue decodepsbt(const JSONRPCRequest& request)
         outputs.push_back(out);
 
         // Fee calculation
-        output_value += psbtx.tx->vout[i].nValue;
+        if (MoneyRange(output_value) && MoneyRange(psbtx.tx->vout[i].nValue) && MoneyRange(output_value + psbtx.tx->vout[i].nValue)) {
```

Comment:
e769f25031a63c378f2efeb957b90a72a8524a5e

Unnecessary check `MoneyRange(output_value)` - starts with zero and next value is already checked. Same with `total_in`.

Reply:
Done